CMakeLists.txt

```diff
@@ -50,6 +50,9 @@ message(STATUS "  ASTARTE_MESSAGE_HUB_PROTO_DIR:   ${ASTARTE_MESSAGE_HUB_PROTO_D
 message(STATUS "--------------------------------------------------")
 
 # Logging library
+# tell spdlog to use an external fmt library instead of its bundled one.
+# useful to have an updated version of fmtlib rather than the deafult one in the system (which can be old)
+set(SPDLOG_FMT_EXTERNAL ON)
```

Comment:
The bundled version of fmt used by spdlog is bundled with the library. The system one is not used unless you set this flag to on.
Additionally the `SPDLOG_USE_STD_FORMAT` and `SPDLOG_FMT_EXTERNAL` options are mutually exclusive and setting both of them results in a compilation error. See [source code of spdlog](https://github.com/gabime/spdlog/blob/9ecdf5c8a1a3ace5673f9fd1857da1b6375b5114/CMakeLists.txt#L106).